app/src/lib/shells/darwin.ts

```diff
@@ -47,11 +47,24 @@ function getBundleID(shell: Shell): string {
   }
 }
 
+async function getOldAlacrittyShellPath(): Promise<string | null> {
+  const oldBundleId = 'in.alacritty'
```

Comment:
@sukhiboi The bundle ID must be `io.alacritty`.

Reply:
right. my mistake. lemme update it.